server/server.js

```diff
@@ -92,12 +93,9 @@ app.get('/keywords', async (req, res) => {
     let query = req.query.search;
     if (query !== undefined) {
         let results = [];
-        let matchOR = query.match(/\w+\/\w+/g);
-        if (matchOR != null) {
-            // search here
-            results = searchWithOr(query, matchOR, searchWithOr, results);
-        }
 
+        results = searchWithOr(query, searchWithOr, results);
+        results = searchWithOptional(query, results);
```

Comment:
I assume that this does not override the other results, but adds to them if we have any??
Does that mean that we are running every single possible search on each query even if there is not the identifier in there?

Reply:
yeah it adds to the results, but there's a check inside to just return back the result if it doesn't match the regex

Reply:
awesome!